src/client/pages/create.js

```diff
@@ -223,13 +239,18 @@ class Create extends React.Component {
               <tr key={i}>
                 <td>{this.state.names[i]}</td>
                 <td>
-                  <a href={`${window.location.origin}/${this.state.gameID}/${i}/${this.state.secret[i]}`} target="_blank" rel="noopener noreferrer">{window.location.origin}/{this.state.gameID}/{i}/{this.state.secret[i]}</a>
+                  <a href={`${this.url(i)}`} target="_blank" rel="noopener noreferrer">{window.location.origin}/{this.state.gameID}/{i}/{this.state.secret[i]}</a>
```

Comment:
We should also use `url` in this line.

Reply:
fixed it and did a search to make sure there weren't any others that I missed